storage-operator/deploy/role.yaml

```diff
@@ -4,36 +4,23 @@ metadata:
   creationTimestamp: null
   name: storage-operator
 rules:
+# For recording transition events
```

Comment:
Good idea to add comments in this file :+1:

Reply:
FWIW, if we update `operator-sdk`/`controller-runtime` (or move to `kubebuilder`), the RBAC rules are to be defined in the Go code using annotations, and this file is to be generated by the tooling.

Reply:
Oh yeah... I'll open a debt ticket for tracking this.